### PR TITLE
Fix some errors in guestbook-go

### PR DESCRIPTION
**Pull request by hw-qiaolei**
Fix some errors in guestbook-go README.md:
1. fix some markdown errors by removing the `<nop>` tag
2. replace some (not all of them) `containers` with `pods`
3. `gcloud comput` -> `gcloud compute`
4. improved sentences that has `list all` to make the descriptions more accurate
5. other tiny fixes

### CODE REVIEW

File: examples/guestbook-go/README.md [hunk 1]
```diff
@@ -59,14 +59,14 @@ This example assumes that you have a working cluster. See the [Getting Started G
...
-<nop>1. Use the [redis-master-controller.json](redis-master-controller.json) file to create the Redis master replication controller in your Kubernetes cluster by running the `kubectl create -f` *`filename`* command:
+1. Use the [redis-master-controller.json](redis-master-controller.json) file to create the Redis master replication controller in your Kubernetes cluster by running the `kubectl create -f` *`filename`* command:
 
     ```console
     $ kubectl create -f examples/guestbook-go/redis-master-controller.json
     replicationcontrollers/redis-master
     ```
 
-<nop>2. To verify that the redis-master-controller is up, list all the replication controllers in the cluster with the `kubectl get rc` command:
+2. To verify that the redis-master controller is up, list all the replication controllers in the cluster(in `default` namespace) with the `kubectl get rc` command:
```
Comment (ashcrow) [inline examples/guestbook-go/README.md]: nit: How about ...```To verify that the redis-master controller is up, list all the replication controllers in the cluster's `default` namespace with the `kubectl get rc` command:```? I think it reads a bit easier.

**Comment by ashcrow** [comment]
One nit added. Otherwise looks good!

File: examples/guestbook-go/README.md (hunk 1 above)
Comment (hw-qiaolei) [inline examples/guestbook-go/README.md]: @ashcrow thanks for reviewing this.
At first I modified the sentence almost the same as you suggested here, but I changed my mind for the following reasons:
1. Explicitly put the *in the cluster's `default` namespace* in the description gives the readers a hint that they should specify `--namespace=default` in the command, which is not the purpose of `default`.
2. Users may change these resources' namespace.

Comment (hw-qiaolei) [inline examples/guestbook-go/README.md]: How about this:
(I put `you created` in the sentence which has a `list all`, and an enhanced comment only in the *first* sentence)

```
list all the replication controllers you created in the cluster with the `kubectl get rc` command(if you don't specify a `--namespace` flag, the `default` namespace will be used. The same below):
...

list all the pods you created in cluster with the `kubectl get pods` command:
...

list all the services you created in the cluster with the `kubectl get services` command:
```

**Comment by hw-qiaolei** [comment]
Updated regarding to @ashcrow 's notes, please check.

File: examples/guestbook-go/README.md [hunk 2]
```diff
@@ -59,14 +59,14 @@ This example assumes that you have a working cluster. See the [Getting Started G
...
-<nop>1. Use the [redis-master-controller.json](redis-master-controller.json) file to create the Redis master replication controller in your Kubernetes cluster by running the `kubectl create -f` *`filename`* command:
+1. Use the [redis-master-controller.json](redis-master-controller.json) file to create the Redis master replication controller in your Kubernetes cluster by running the `kubectl create -f` *`filename`* command:
 
     ```console
     $ kubectl create -f examples/guestbook-go/redis-master-controller.json
     replicationcontrollers/redis-master
     ```
 
-<nop>2. To verify that the redis-master-controller is up, list all the replication controllers in the cluster with the `kubectl get rc` command:
+2. To verify that the redis-master controller is up, list the replication controllers you created in the cluster with the `kubectl get rc` command(if you don't specify a `--namespace`, the `default` namespace will be used. The same below):
```
Comment (ashcrow) [inline examples/guestbook-go/README.md]: "the same below" could be expanded to "Please note that the same applies to all following verification commands" or something for explicit like that. However, I think this is still much better than what was there before so I won't even consider it a nit :smile:

**Comment by ashcrow** [comment]
@hw-qiaolei makes sense. This looks like a good documentation improvement to me.

**Comment by hw-qiaolei** [comment]
Hello @bgrant0607, would you please review this PR? This is about the doc and quite easy to check.

**Comment by bgrant0607** [comment]
LGTM, thanks.

**Pull request merged**